modules/core/04-channel/v2/keeper/relay.go

```diff
@@ -107,15 +113,9 @@ func (k Keeper) recvPacket(
 	// that the packet was indeed sent by our counterparty.
 	counterparty, ok := k.GetCounterparty(ctx, packet.DestinationId)
 	if !ok {
-		// If the counterparty is not found, attempt to retrieve a v1 channel from the channel keeper
-		// if it exists, then we will convert it to a v2 counterparty and store it in the packet server keeper
-		// for future use.
 		// TODO: figure out how aliasing will work when more than one packet data is sent.
-		if counterparty, ok = k.AliasV1Channel(ctx, packet.Data[0].SourcePort, packet.SourceId); ok {
-			// we can key on just the source channel here since channel ids are globally unique
-			k.SetCounterparty(ctx, packet.SourceId, counterparty)
-		} else {
-			// if neither a counterparty nor channel is found then simply return an error
+		counterparty, ok = k.getV1Counterparty(ctx, packet.Data[0].SourcePort, packet.SourceId)
```

Comment:
Should this be destination port/destination id? (on recv side, line 114 also uses destination id

Reply:
ouch yeah thanks this slipped previous reviews too! Thanks